src/find_library.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include "rcpputils/find_library.hpp"
 
+#include <link.h>
```

Comment:
You'll need to add an ifdef around this as this does not exist on Apple/Windows.

Reply:
Sure, will fix